widgetssdk/src/main/java/com/glia/widgets/core/fileupload/GliaFileProvider.java

```diff
@@ -2,5 +2,13 @@
 
 import androidx.core.content.FileProvider;
 
+/**
+ * Glia internal class.
+ *
+ * It will be automatically added to the integrator's manifest file by the manifest merger during compilation.
+ *
+ * This custom FileProvider is used to prevent confusion with requests to the default FileProvider that may be used by the integrator.
```

Comment:
```suggestion
 * This custom `FileProvider` is used to prevent confusion with requests to the default `FileProvider` that may be used by the integrator.
```